bigchaindb/common/schema/transaction.yaml

```diff
@@ -0,0 +1,256 @@
+---
+"$schema": "http://json-schema.org/draft-04/schema#"
+id: "http://www.bigchaindb.com/schemas/transaction.json"
+type: object
+additionalProperties: false
+title: Transaction Schema
+description: |
+    This is the outer transaction wrapper. It contains the ID, version and the body of the transaction, which is also called `transaction`.
+required:
+- id
+- transaction
+- version
+properties:
+  id:
+    "$ref": "#/definitions/sha3_hexdigest"
+    description: |
+        A sha3 digest of the transaction. The ID is calculated by removing all
+        derived hashes and signatures from the transaction, serializing it to
+        JSON with keys in sorted order and then hashing the resulting string
+        with sha3.
+  transaction:
+    type: object
+    title: transaction
+    description: |
+        Body of the transaction.
```

Comment:
Below, you call this thing the "Transaction Body" so for consistency it would be nice to use the same terminology here. Is it possible to hyperlink from here to that section below?